oam/install_scripts/columnstore-post-install.in

```diff
@@ -174,36 +204,39 @@ mkdir $lockdir >/dev/null 2>&1
 
 rm -f $lockdir/columnstore
 
-#check and get amazon env variables
-aws=`which aws 2>/dev/null`
-if [ -z "aws" ]; then
-    MCSgetCredentials.sh >/dev/null 2>&1
+# This was the last place of postConfigure. RIP
+
+/usr/sbin/install_mcs_mysql.sh --tmpdir=$tmpDir
+
+# Restart MDB to enable plugin
+systemctl cat mariadb.service > /dev/null 2>&1
+if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
+    systemctl restart mariadb.service > /dev/null 2>&1
+else
+    pkill mysqld > /dev/null 2>&1
+    /usr/bin/mysqld_safe &
 fi
 
-postConfigure
```

Comment:
post-mysqld-install and post-mysql-install would be called during postConfigure. is this working properly? i see install_mcs_mysql.sh and the checkforerror, but where is mysql_install_db which was previously called inside post-mysqld-install?

Reply:
Hm, I though my email comment should get here but it seems got lost.
mysql_install_db generates initial MDB syscat and we don't need this anymore.